src/Tools/Test Framework/Test Runner/src/TestRunnerIsolDisabled.Codeunit.al

```diff
@@ -14,33 +14,37 @@ codeunit 130451 "Test Runner - Isol. Disabled"
                   tabledata "Test Method Line" = rimd;
 
     trigger OnRun()
+    var
+        ALTestSuite: Record "AL Test Suite";
     begin
         if Rec."Skip Logging Results" then
-            TestRunnerMgt.RunTestsWithoutLoggingResults(Rec)
+            this.TestRunnerMgt.RunTestsWithoutLoggingResults(Rec)
         else begin
-            ALTestSuite.Get(Rec."Test Suite");
-            CurrentTestMethodLine.Copy(Rec);
-            TestRunnerMgt.RunTests(Rec);
+            if ALTestSuite.Get(Rec."Test Suite") then
```

Comment:
Add SetLoadFields

Reply:
No gain from doing this. It is a small table, no table extensions.
Now the table extensions are merged into a single table so gain is even more questionable.